app/uk/gov/hmrc/gform/fileupload/FileUploadService.scala

```diff
@@ -95,6 +96,17 @@ class FileUploadService(
 
   def deleteFile(envelopeId: EnvelopeId, fileId: FileId)(implicit hc: HeaderCarrier): Future[Unit] =
     fileUploadConnector.deleteFile(envelopeId, fileId)
+
+  def uploadAttachment(envelopeId: EnvelopeId, fileAttachment: FileAttachment)(
+    implicit hc: HeaderCarrier): Future[Unit] =
+    fileUploadFrontendConnector.upload(
+      envelopeId,
+      FileId("anything"),
```

Comment:
Shouldn't `fileId` be unique?

Reply:
Yep, I think so. It looks like the hardcoded values e.g FileId("pdf") etc are there in the scenarios where only one file needs to be sent so there is no need to generate unique values. I will just generate a random uuid string and set that as the value.

Reply:
fixed